packages/protoc-gen-connect-query-react/src/generateDts.ts

```diff
@@ -28,12 +32,34 @@ const { safeIdentifier } = codegenInfo;
  */
 const generateServiceFile =
   (schema: Schema, protoFile: DescFile) => (service: DescService) => {
+    const { MethodKind: rtMethodKind, MethodIdempotency: rtMethodIdempotency } = schema.runtime;
 
     const f = schema.generateFile(
         `${protoFile.name}-${localName(service)}_connectquery_react.d.ts`,
     );
     f.preamble(protoFile);
 
+    f.print(makeJsDoc(service));
+    f.print("export declare const ", localName(service), ": {");
```

Comment:
Added service definition generation here so we can use that for tests directly without needing to build it or rely on connect-es generation as well.